### PR TITLE
Patch 20180315

### PR DESCRIPTION
**Pull request by AndyHee**
I rephrased this, as it was rather convoluted and contained culturally specific instructions that may not be generally applicable.

### CODE REVIEW

File: nsfw/nsfw.php [hunk 1]
```diff
@@ -74,15 +74,15 @@ function nsfw_addon_settings(&$a, &$s)
 	}
 
 	$s .= '<span id="settings_nsfw_inflated" class="settings-block fakelink" style="display: block;" onclick="openClose(\'settings_nsfw_expanded\'); openClose(\'settings_nsfw_inflated\');">';
-	$s .= '<h3>' . L10n::t('Not Safe For Work (General Purpose Content Filter)') . '</h3>';
+	$s .= '<h3>' . L10n::t('Content Filter (NSFW tag and more)') . '</h3>';
```
Comment (tobiasd) [inline nsfw/nsfw.php]: You should drop the "tag" after NSFW I think

**Comment by tobiasd** [comment]
Also, could you run

    ../path/to//friendica/util/run_xgettext.sh -s

in the addon directory to regenerate the `lang/C/messages.po` file so transifex can pick up the new strings. If not, just let me know and do it after the merge.

**Comment by AndyHee** [comment]
Thanks Tobias, I'll change that.

That's the command I was looking for. :-)

File: nsfw/nsfw.php (hunk 1 above)
Comment (AndyHee) [reply nsfw/nsfw.php]: Will do.

**Comment by AndyHee** [comment]
Hypolite should be acknowledged for his help on my previous unsuccessful attempts to change these files

**Pull request merged**